README.Rmd

```diff
@@ -13,27 +15,24 @@ knitr::opts_chunk$set(
   eval = FALSE,
   comment = "#>"
 )
-library(knitr)
 ```
 
 Create purls from the filtered sands of your dependencies, powered by OSS Index
 
 ## Usage
 
-If this project is run independently, one can run:
+The package has two main functions. The first, function extracts
+your installed R packages and uses the sonatype API to scan for 
```

Comment:
Would do `Sonatype OSS Index API`, otherwise I can already see an email in my inbox from someone about branding!

Reply:
Done